src/OmniSharp.Cake/Services/RequestHandlers/CakeRequestHandler.cs

```diff
@@ -54,11 +54,16 @@ protected CakeRequestHandler(OmniSharpWorkspace workspace)
 
             request = await TranslateRequestAsync(request);
 
-            var response = await service.Handle(request);
+            var response = await HandleCore(request, service);
 
             return await TranslateResponse(response, request);
         }
 
+        public virtual  Task<TResponse> HandleCore(TRequest request, IRequestHandler<TRequest, TResponse> service)
```

Comment:
Is it necessary to introduce this? `Task<TResponse> Handle(TRequest request)` is already `virtual`.